### PR TITLE
Release 4.3.0

### PR DESCRIPTION
**Pull request by ehdsouza**
This PR is generated using:

API Definitions: `sdk-2020-02-10`

Open API SDK generator: `2.2.4`

### CODE REVIEW

**Comment by germanattanasio** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 10
           

Clones added
============
- ibm_watson/visual_recognition_v4.py  5
- test/unit/test_visual_recognition_v4.py  26
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724&bid=16485235)

File: ibm_watson/tone_analyzer_v3.py [hunk 1]
```diff
@@ -30,7 +30,8 @@
 from .common import get_sdk_headers
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
-from ibm_cloud_sdk_core import get_authenticator_from_environment
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/tone_analyzer_v3.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/visual_recognition_v4.py [hunk 2]
```diff
@@ -24,11 +24,13 @@
 from datetime import datetime
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/visual_recognition_v4.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/personality_insights_v3.py [hunk 3]
```diff
@@ -38,7 +38,8 @@
 from .common import get_sdk_headers
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
-from ibm_cloud_sdk_core import get_authenticator_from_environment
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/personality_insights_v3.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/language_translator_v3.py [hunk 4]
```diff
@@ -27,8 +27,9 @@
 from datetime import datetime
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/language_translator_v3.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/natural_language_classifier_v1.py [hunk 5]
```diff
@@ -26,8 +26,9 @@
 from datetime import datetime
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/natural_language_classifier_v1.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/visual_recognition_v4.py [hunk 6]
```diff
@@ -24,11 +24,13 @@
...
 from ibm_cloud_sdk_core import datetime_to_string, string_to_datetime
-from ibm_cloud_sdk_core import get_authenticator_from_environment
+from ibm_cloud_sdk_core.get_authenticator import get_authenticator_from_environment
 from typing import BinaryIO
 from typing import Dict
 from typing import List
+from typing import TextIO
```
Comment (germanattanasio) [inline ibm_watson/visual_recognition_v4.py]: Codacy Issue found: [Unused TextIO imported from typing](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/visual_recognition_v3.py [hunk 7]
```diff
@@ -25,8 +25,9 @@
 from datetime import datetime
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/visual_recognition_v3.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/assistant_v2.py [hunk 8]
```diff
@@ -26,7 +26,8 @@
 from .common import get_sdk_headers
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
-from ibm_cloud_sdk_core import get_authenticator_from_environment
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/assistant_v2.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: ibm_watson/text_to_speech_v1.py [hunk 9]
```diff
@@ -35,7 +35,8 @@
 from .common import get_sdk_headers
 from enum import Enum
 from ibm_cloud_sdk_core import BaseService
-from ibm_cloud_sdk_core import get_authenticator_from_environment
+from ibm_cloud_sdk_core import DetailedResponse
```
Comment (germanattanasio) [inline ibm_watson/text_to_speech_v1.py]: Codacy Issue found: [Unused DetailedResponse imported from ibm_cloud_sdk_core](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: test/integration/test_language_translator_v3.py [hunk 10]
```diff
@@ -5,39 +5,42 @@
...
         assert document_id is not None
 
-        document_status = self.language_translator.get_document_status(document_id).get_result()
+        document_status = self.language_translator.get_document_status(
+            document_id).get_result()
         assert document_status is not None
 
         if document_status.get('status') == 'available':
-            response = self.language_translator.get_translated_document(document_id, 'text/plain').get_result()
+            response = self.language_translator.get_translated_document(
```
Comment (germanattanasio) [inline test/integration/test_language_translator_v3.py]: Codacy Issue found: [Too many positional arguments for method call](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)

File: examples/assistant_tone_analyzer_integration/tone_detection.py [hunk 11]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 IBM All Rights Reserved.
+# (C) Copyright IBM Corp. 2016, 2019.
```
Comment (mediumTaj) [inline examples/assistant_tone_analyzer_integration/tone_detection.py]: 👍

**Comment by mediumTaj** [comment]
Please don't forget to add examples https://github.ibm.com/Watson/developer-experience/wiki/SDK-Regular-Release-%232-2020-Examples

**Pull request merged**